mne/io/nsx/nsx.py

```diff
@@ -0,0 +1,601 @@
+# Author: Porloy Das <pdas6@mgh.harvard.edu>
+#
+# License: BSD-3-Clause
+import os
+from datetime import datetime, timezone
+from warnings import warn
+
+import numpy as np
+
+from ..constants import FIFF
+from ..meas_info import _empty_info
+from ..base import BaseRaw
+from ..utils import _read_segments_file
+
+from ...annotations import Annotations
+from ...utils import logger, fill_doc
+
+
+# common channel type names mapped to internal ch types
+CH_TYPE_MAPPING = {
+    "EEG": FIFF.FIFFV_EEG_CH,
+    "SEEG": FIFF.FIFFV_SEEG_CH,
+    "CC": FIFF.FIFFV_SEEG_CH,
+    "ECOG": FIFF.FIFFV_ECOG_CH,
+    "DBS": FIFF.FIFFV_DBS_CH,
+    "EOG": FIFF.FIFFV_EOG_CH,
+    "ECG": FIFF.FIFFV_ECG_CH,
+    "EMG": FIFF.FIFFV_EMG_CH,
+    "BIO": FIFF.FIFFV_BIO_CH,
+    "RESP": FIFF.FIFFV_RESP_CH,
+    "TEMP": FIFF.FIFFV_TEMPERATURE_CH,
+    "MISC": FIFF.FIFFV_MISC_CH,
+    "SAO2": FIFF.FIFFV_BIO_CH,
+}
```

Comment:
To reduce duplication I would have done just the exceptions:
```suggestion
CH_TYPE_MAPPING = {
    "CC": "SEEG",
    "TEMP": "TEMPERATURE_CH",
    "SAO2": "BIO",
}
```
Then later any use would be something like
```
ch_const = getattr(FIFF, f'FIFFV_{_CH_TYPE_MAPPING.get(ch_type, ch_type)}_CH')
```
but if you use it a lot maybe it's not so good. And what you already have is okay. So up to you :shrug: